### DISCUSSION TITLE
Deploy doks on Azure Static Web Apps - Issues with PostCSS

### DISCUSSION THREAD

**Discussion by LukeEvansTech** [thuliteio/doks#828] [Q&A]
Hi all,

Was wondering if anyone has successfully deployed this to an Azure Static Web App?

I used the GitHub template feature from https://github.com/h-enk/doks.git. 

I haven't changed any files as of yet to just test the deployment mechanism. I can build the same site locally and deploy to Netlify without issue from the same repo.

The Azure Static Web App deployment fails with the following:

`Start building sites … `
`hugo v0.101.0-466fa43c16709b4483689930a4f9ac8add5c9f66+extended linux/amd64 BuildDate=2022-06-16T07:09:16Z VendorInfo=gohugoio`
`Error: Error building site: POSTCSS: failed to transform "main.css" (text/css): SystemError [ERR_SYSTEM_ERROR]: A system error occurred: uv_os_homedir returned ENOENT (no such file or directory)`
    `at Object.<anonymous> (/opt/nodejs/17.6.0/lib/node_modules/npm/node_modules/clean-stack/index.js:6:61)`
   ` at Module._compile (node:internal/modules/cjs/loader:1097:14)`
    `at Object.Module._extensions..js (node:internal/modules/cjs/loader:1151:10)`
   ` at Module.load (node:internal/modules/cjs/loader:975:32)`
`Total in 1616 ms`
`    at Function.Module._load (node:internal/modules/cjs/loader:822:12)`
`    at Module.require (node:internal/modules/cjs/loader:999:19)`
`    at require (node:internal/modules/cjs/helpers:102:18)`
`    at Object.<anonymous> (/opt/nodejs/17.6.0/lib/node_modules/npm/node_modules/aggregate-error/index.js:3:20)`
`    at Module._compile (node:internal/modules/cjs/loader:1097:14)`
`    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1151:10)`

Would appreciate any help!

**Comment by dagholmus** [comment]
@LukeEvansTech Did you find a solution on this? I have the exact same problem.

Build is successful both locally and to Netlify, but it fails when deploying to Azure Static Web Apps. I have tested with a lot of different configurations, environment variables, setups ++.
